### ISSUE TITLE
Test Failure: junit.framework.TestSuite.com.ibm.ws.request.timing.monitor.fat.RequestTimingMetricsTest

### ISSUE THREAD

**Issue by fmhwong** [OpenLiberty/open-liberty#30073]
RTC 302667
```
com.ibm.ws.request.timing.monitor.fat.RequestTimingMetricsTest:java.lang.Exception: Runtime features were not of the expected version for repeat action (Server: RequestTimingFeatureWithMetrics, Action: EE8_FEATURES_MicroProfile_33).
Expected: servlet-4.0, Actual: servlet-3.1.
Expected: cdi-2.0, Actual: cdi-1.2.
This is usually caused by a feature not being explicitly set in the FAT's server.xml such that FeatureReplacementAction does not replace it properly.
    at componenttest.topology.impl.LibertyServer.checkServerRepeatFeatures(LibertyServer.java:3640)
    at componenttest.topology.impl.LibertyServer.stopServer(LibertyServer.java:3301)
    at componenttest.topology.impl.LibertyServer.stopServer(LibertyServer.java:3143)
    at componenttest.topology.impl.LibertyServer.stopServer(LibertyServer.java:3137)
    at componenttest.topology.impl.LibertyServer.stopServer(LibertyServer.java:3113)
    at componenttest.topology.impl.LibertyServer.stopServer(LibertyServer.java:3022)
    at com.ibm.ws.request.timing.monitor.fat.RequestTimingMetricsTest.classTearDown(RequestTimingMetricsTest.java:144)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
    at componenttest.rules.repeater.RepeatTests$CompositeRepeatTestActionStatement.evaluate(RepeatTests.java:149)
    at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:374)
    at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:178)
```

RTC 302665
```
testVendorMetricsPresent_EE11_FEATURES_MicroProfile_70:java.lang.Exception: 2024-11-04-14:28:04:159 Runtime features were not of the expected version for repeat action (Server: MetricsMonitorServer, Action: EE11_FEATURES_MicroProfile_70).
Expected: servlet-6.1, Actual: servlet-6.0.
This is usually caused by a feature not being explicitly set in the FAT's server.xml such that FeatureReplacementAction does not replace it properly.
    at componenttest.topology.impl.LibertyServer.checkServerRepeatFeatures(LibertyServer.java:3640)
    at componenttest.topology.impl.LibertyServer.stopServer(LibertyServer.java:3301)
    at componenttest.topology.impl.LibertyServer.stopServer(LibertyServer.java:3143)
    at componenttest.topology.impl.LibertyServer.stopServer(LibertyServer.java:3137)
    at componenttest.topology.impl.LibertyServer.stopServer(LibertyServer.java:3113)
    at componenttest.topology.impl.LibertyServer.stopServer(LibertyServer.java:3022)
    at componenttest.topology.impl.LibertyServer.stopServer(LibertyServer.java:2997)
    at io.openliberty.microprofile.metrics.internal.monitor_fat.MetricsMonitorTest.tearDown(MetricsMonitorTest.java:94)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
    at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:209)
    at componenttest.rules.repeater.RepeatTests$CompositeRepeatTestActionStatement.evaluate(RepeatTests.java:149)
    at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:374)
    at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:178)
```

**Comment by jhanders34** [comment]
https://github.com/OpenLiberty/open-liberty/pull/30063 was temporarily done by @tevans78 to not check the servers until this can be updated in the test to include servlet in the server.xml likely.

**Comment by pgunapal** [comment]
RTC 302667 fixed in #30103

Keeping this open, to address RTC 302665

**Comment by pgunapal** [comment]
Both RTC defects addressed.
